macros/get_ticket_tag_columns.sql

```diff
@@ -0,0 +1,11 @@
+{% macro get_ticket_tag_columns() %}
+
+{% set columns = [
+    {"name": "_fivetran_synced", "datatype": dbt_utils.type_timestamp()},
+    {"name": "'tag'", "datatype": dbt_utils.type_string(), "alias": "tags"},
```

Comment:
While we typically alias fields within the staging model, this was necessary as tag is a reserved word within redshift. 

@fivetran-kristin @fivetran-jamie are there other ways we have done this in past packages? If so, should we update this to be consistent with how we have worked with reserved words in the past?

Reply:
Updated this based off Jamie's documentation in the solutions core process doc to alias within the staging model and not the macro.